counter/Counter.js

```diff
@@ -0,0 +1,69 @@
+class Counter {
+	static #counterTree = {
+		type: 'div',
+		cssClasses: ['counter'],
+		children: [
+			{
+				type: 'h1',
+				cssClasses: ['counter__heading'],
+				children: [
+					{ type: 'text', value: 'Click count: ' },
+					{
+						type: 'span',
+						cssClasses: ['counter__value'],
+						attributes: [{ name: 'data-counter-polarity', value: 'zero' }],
+						children: [{ type: 'text', value: '0' }]
+					}
+				]
+			},
+			{
+				type: 'dev',
```

Comment:
Кажется, тут опечатка
```suggestion
				type: 'div',
```